requirements.txt

```diff
@@ -8,3 +8,4 @@ ply==3.4
 slimit==0.8.1
 six==1.9.0
 django-appconf==1.0.1
+python-dateutil==2.4.2
```

Comment:
[See here, for example](https://github.com/learningequality/ka-lite/blob/develop/kalite/control_panel/views.py#L4). It was a missing dependency for me -- could be my distribution of python? But it doesn't seem to be a standard library module.